### ISSUE TITLE
Project abandoned?

### ISSUE THREAD

**Issue by rotrer** [strebl/ldap-auth#29]
Hi!,

Why composer says: "Package strebl/adldap is abandoned, you should avoid using it. Use adldap2/adldap2 instead", after install.

Specs:
- Laravel 5.1
- Composer: "strebl/l5-ldap-auth": "2.1.*"

**Comment by strebl** [comment]
That's true. The maintainer of the ldap-auth project (ccovey) stopped maintaining it. That's also true for the adldap dependency. I just forked the package to provide Laravel 5 support. In the meantime @stevebauman created the Adldap successor [Adldap2](https://github.com/Adldap2/Adldap2). Part of this project is the [Laravel package](https://github.com/Adldap2/Adldap2-Laravel) which is better documented and coded and also has more features.
@stevebauman puts so much time in this project and is a great developer.
That's the reason I abandoned my fork and recommend people to switch to the the new repo.

Did that answer your question?
